samples/src/Microsoft.Azure.IIoT.App/src/Models/PublisheInfoRequested.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.App.Models {
+    using System;
+
+    public class PublisheInfoRequested {
+
+        public PublisheInfoRequested(PublisherInfo publisher) {
+            _requestedMaxWorkers = publisher?.PublisherModel?.Configuration?.MaxWorkers;
+            _requestedHeartbeatInterval = publisher?.PublisherModel?.Configuration?.HeartbeatInterval;
+            _requestedJobCheckInterval = publisher?.PublisherModel?.Configuration?.JobCheckInterval;
```

Comment:
What is heartbeat interval used for?

Reply:
It's the interval used by publisher to calls in the orchestrator to report status and ask for job changes.

Reply:
Thanks